wire-ios/Wire-iOS/Resources/de.lproj/Localizable.strings

```diff
@@ -1117,7 +1116,6 @@
 "self.settings.account_picture_group.color" = "Profilfarbe";
 "self.settings.account_picture_group.accent_color.blue" = "Blau";
 "self.settings.account_picture_group.accent_color.green" = "Grün";
-"self.settings.account_picture_group.accent_color.yellow" = "Gelb";
```

Comment:
@caldrian did we remove this ?

Reply:
Yes, yellow was marked as deprecated and not used anymore before I joined. I cleaned it up now.